text/2021-12-09-logging.md

```diff
@@ -209,12 +209,30 @@ func NewWebModule(logger logr.Logger) *WebModule {
 }
 ```
 
+The name of the logger should follow these rules:
+
+- use kebab-case for the name
+- follow pattern: `<application-name>.<component-name>.<component-name>`
+
```

Comment:
Do we need to be case-sensitive? For example, all lower-case?

Reply:
"kebab-case" requires using lower-cases :)

Reply:
Get!